boards/arm/stm32h7/weact-stm32h743/src/weact-stm32h743.h

```diff
@@ -36,6 +36,13 @@
 
 /* Configuration ************************************************************/
 
+#define HAVE_PROC            0
+#define HAVE_USBDEV          1
+#define HAVE_USBHOST         1
+#define HAVE_USBMONITOR      0
+#define HAVE_MTDCONFIG       0
+#define HAVE_PROGMEM_CHARDEV 0
```

Comment:
Normally all HAVE_xxx should be defined as 1 and undefined case some criteria (i.e.: CONFIG_ symbol missing) is not met